tracpro/templates/baseline/baselineterm_header.html

```diff
@@ -4,26 +4,47 @@
 <h2 class="page-header">
   {{ baselineterm.name }}
 </h2>
-
 <div class="clearfix">
-  {# if org_perms.contacts.contact_update or org_perms.contacts.contact_delete #}
+  <div class="pull-back">
+    <form class='form-inline' method='get'>
+      <label class='control-label' for='region-select'></label>
+      {% trans "Filter By Region" %}
+      <select id='region-select' class='form-control' onchange='onChartRegionChange(this)' name='region'>
+        <option value='0'>All Regions</option>
+        {% for region in all_regions %}
+          {% for key, value in region.items %}
+              {% if key == "response__contact__region" %}
+                  {% if value == region_selected %}
+                    <option selected='selected' value='{{ value }}'>
+                  {% else %}
+                    <option value='{{ value }}'>
+                  {% endif %}
+              {% else %}
+                  {{ value }}</option>
```

Comment:
I'm having trouble seeing how this `</option>` can match up to an opening `<option>` tag.